localstack/services/edge.py

```diff
@@ -388,11 +389,13 @@ def ensure_can_use_sudo():
         run("sudo -v", stdin=True)
 
 
-def start_component(component: str, port=None, use_ssl=True, asynchronous=False):
+def start_component(component: str, port=None):
```

Comment:
just a general note: i'm typically very careful with removing kwargs from legacy code. things sometimes break in unexpected ways :grimacing: obviously i haven't looked into this particular case and maybe it's safe. but since it's code we're going to remove soon anyway, it's safer to just grit your teeth and not change things that don't need changing.

Reply:
I checked quite carefully, `start_component` is really only used in the main of `edge.py`.
I just removed these because they were somewhat confusing (the default params were used always anyways, however, the different components take different params, the DNS component f.e. does not take the `use_ssl`).